src/components/emptyStatePages/EmptyStatePage.tsx

```diff
@@ -296,6 +309,23 @@ export default function EmptyStatePage({
           />
         </>
       )}
+      <>
+        <PlantingSiteSelectTypeModal
+          open={plantingSiteTypeModalOpen}
+          onNext={(goToCreate) => {
+            if (goToCreate) {
+              goTo(APP_PATHS.PLANTING_SITES_NEW);
+            } else {
+              setPlantingSiteTypeHelpModalOpen(true);
+            }
+          }}
+          onClose={() => setPlantingSiteTypeModalOpen(false)}
+        />
+        <PlantingSiteWithMapHelpModal
+          open={plantingSiteTypeHelpModalOpen}
+          onClose={() => setPlantingSiteTypeHelpModalOpen(false)}
+        />
+      </>
```

Comment:
extract out as a component for reuse in both places?

Reply:
That'd be less repetition, so I'll make that change. Thanks